src/modules/job-manager/journal.c

```diff
@@ -20,7 +20,9 @@
 #include "src/common/libczmqcontainers/czmq_containers.h"
 #include "src/common/libeventlog/eventlog.h"
 #include "src/common/libutil/errno_safe.h"
+#include "src/common/libutil/errprintf.h"
```

Comment:
typo (double "now") in commit message for 18bb02cad4cf372ab27b481a5e6c9e0f82a45680:

>  but now job-manager now supports